python/rpdk/python/codegen.py

```diff
@@ -339,6 +347,8 @@ def _docker_build(cls, external_path):
             cause.__cause__ = e
             raise DownstreamError("Error running docker build") from cause
         except (ContainerError, ImageLoadError, APIError) as e:
+            for line in logs:
```

Comment:
Similar to 344.  Function?

Reply:
Just for debugging.  Wanted to capture odd usecases in the logs, strange mismatches in pip packages etc.